src/main/java/gov/cabinetoffice/gap/adminbackend/security/WebSecurityConfig.java

```diff
@@ -23,7 +23,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         http.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.IF_REQUIRED).and()
                 .authorizeHttpRequests(
                         auth -> auth
-                                .mvcMatchers("/login", "/health", "/emails/sendLambdaConfirmationEmail",
+                                .mvcMatchers("/v2/login", "/login", "/health", "/emails/sendLambdaConfirmationEmail",
```

Comment:
I'm confused - is this saying you must be authenticated to login? 🤣

Reply:
Means there must be an auth header, which we are using here:

```
    UsernamePasswordAuthenticationToken authenticationToken = new UsernamePasswordAuthenticationToken("",
                httpRequest.getHeader(HttpHeaders.AUTHORIZATION));
```

Same as old /login mapping